assets/js/blocks/collection-filters/inner-blocks/stock-filter/frontend.ts

```diff
@@ -43,11 +43,11 @@ interactivityStore( {
 	actions: {
 		filters: {
 			navigate: ( { context }: { context: DropdownContext } ) => {
-				if ( context.woocommerceDropdown.selectedItem.value ) {
-					navigate(
-						getUrl( context.woocommerceDropdown.selectedItem.value )
-					);
-				}
+				navigate(
+					getUrl(
+						context.woocommerceDropdown.selectedItem.value || ''
+					)
+				);
 			},
 			updateProducts: ( { event }: ActionProps ) => {
```

Comment:
I think we should, for now, rename this action to an unique one to avoid collision with others
```suggestion
			updateProductsWithStockFilters: ( { event }: ActionProps ) => {
```

Reply:
Thanks I forgot you mentioned this last time. Will fix.

Reply:
Resolved in https://github.com/woocommerce/woocommerce-blocks/pull/11827/commits/ba0db6b4f486d5762a3909abe9e0f065a2e6955d